includes/class-wc-stripe-webhook-handler.php

```
@@ -120,22 +120,19 @@ public function check_for_webhook() {
 			return;
 		}
 
-		WC_Stripe_Webhook_State::set_pending_webhooks_count( $event->pending_webhooks );
+		WC_Stripe_Webhook_State::set_pending_webhooks_count( $event->pending_webhooks ?? 0 );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Webhook_State' in method 'check_for_webhook'. (undefined)

(StaticAccess)